runtime/src/bank/tests.rs

```diff
@@ -12063,7 +12062,7 @@ fn test_feature_activation_loaded_programs_epoch_transition() {
     let feature_account_balance =
         std::cmp::max(genesis_config.rent.minimum_balance(Feature::size_of()), 1);
     bank.store_account(
-        &feature_set::reject_callx_r10::id(),
+        &feature_set::disable_fees_sysvar::id(),
```

Comment:
why was this necessary?

Reply:
Because the `reject_callx_r10` is cleaned up, so it has no more influence. Had to find any other RBPF related feature.